documentation/modules/configuring/con-partition-reassignment.adoc

```diff
@@ -15,7 +15,12 @@ Cruise Control is recommended for this type of operation as it xref:cruise-contr
 
 Scaling topic replication factor up and down:: Increase or decrease the replication factor of your Kafka topics. To do this, you must create a partition reassignment plan that identifies the existing replication assignment across partitions and an updated assignment with the replication factor changes.
 
-Changing the preferred leader:: Change the preferred leader of a Kafka partition. This can be useful if the current preferred leader is unavailable or if you want to redistribute load across the brokers in the cluster. To do this, you must create a partition reassignment plan that specifies the new preferred leader for each partition by changing the order of replicas.
+Changing the preferred leader:: Change the preferred leader of a Kafka partition. 
+In Kafka, the preferred leader is the replica that serves as the leader for a partition.
+Changing the preferred leader can be useful if the current preferred leader is unavailable or if you want to redistribute load across the brokers in the cluster.
```

Comment:
If the preferred leader is unavailable, another in-sync replica is automatically elected as leader, or the partition goes offline if there are no in-sync replicas. A background thread moves the leader role to the preferred replica when it is in sync. This means that changing the preferred replicas only makes sense in the context of a cluster rebalancing.